### PR TITLE
[doc][yba] add OIDC URI example

### PR DESCRIPTION
**Pull request by ddhodge**
Add OIDC URI example
DOC-418

@netlify /stable/yugabyte-platform/security/authentication/oidc-authentication-aad

### CODE REVIEW

File: docs/content/preview/yugabyte-platform/administer-yugabyte-platform/oidc-authentication.md [hunk 1]
```diff
@@ -94,6 +94,7 @@ You configure OIDC as follows:
     - In the **Scope** field, enter your identity provider OIDC scope that is allowed to be requested. This field accepts a space-separated list of values. If left blank, all scopes will be considered.
     - In the **Email Attribute** field, enter the OIDC scope containing the user email identifier. This field accepts a case-sensitive custom configuration. Typically, this field is left blank.
     - If you have configured OIDC to use [refresh tokens](https://openid.net/specs/openid-connect-core-1_0.html#RefreshTokens), in the **Refresh Token URL** field, enter the URL of the refresh token endpoint.
+    - If you have configured [OIDC enhancements](../security/authentication/oidc-authentication-aad/#enable-oidc-enhancements), you can select the **Display JWT token on login** option to allow users to access their JWT from the YugabyteDB Anywhere sign in page. See [Set up OIDC with Azure AD on YugabyteDB Anywhere](../../security/authentication/oidc-authentication-aad/#set-up-oidc-with-azure-ad-on-yugabytedb-anywhere).
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/administer-yugabyte-platform/oidc-authentication.md:97]:
```suggestion
    - If you have configured [OIDC enhancements](../../security/authentication/oidc-authentication-aad/#enable-oidc-enhancements), you can select the **Display JWT token on login** option to allow users to access their JWT from the YugabyteDB Anywhere sign in page. See [Set up OIDC with Azure AD on YugabyteDB Anywhere](../../security/authentication/oidc-authentication-aad/#set-up-oidc-with-azure-ad-on-yugabytedb-anywhere).
```

**Review by asharma-yb — APPROVED** [review]
Looks good!

File: docs/content/stable/yugabyte-platform/security/authentication/oidc-authentication-aad.md [hunk 2]
```diff
@@ -114,7 +114,11 @@ To register an application, do the following:
 
 1. Select the tenant for the application.
 
-1. Set the redirect URI. This is where the IdP redirects after authentication.
+1. Set the redirect URI. This is where the IdP redirects after authentication. The URI is in the following form:
```
Comment (aishwarya24) [inline docs/content/stable/yugabyte-platform/security/authentication/oidc-authentication-aad.md:117]: should be in preview too right?

File: docs/content/preview/yugabyte-platform/administer-yugabyte-platform/oidc-authentication.md [hunk 3]
```diff
@@ -94,6 +94,7 @@ You configure OIDC as follows:
     - In the **Scope** field, enter your identity provider OIDC scope that is allowed to be requested. This field accepts a space-separated list of values. If left blank, all scopes will be considered.
     - In the **Email Attribute** field, enter the OIDC scope containing the user email identifier. This field accepts a case-sensitive custom configuration. Typically, this field is left blank.
     - If you have configured OIDC to use [refresh tokens](https://openid.net/specs/openid-connect-core-1_0.html#RefreshTokens), in the **Refresh Token URL** field, enter the URL of the refresh token endpoint.
+    - If you have configured [OIDC enhancements](../../security/authentication/oidc-authentication-aad/#enable-oidc-enhancements), you can select the **Display JWT token on login** option to allow users to access their JWT from the YugabyteDB Anywhere sign in page. See [Set up OIDC with Azure AD on YugabyteDB Anywhere](../../security/authentication/oidc-authentication-aad/#set-up-oidc-with-azure-ad-on-yugabytedb-anywhere).
```
Comment (aishwarya24) [inline docs/content/preview/yugabyte-platform/administer-yugabyte-platform/oidc-authentication.md:97]: This in stable ?

**Review by aishwarya24 — APPROVED** [review]
Just a doubt about copying to preview and stable. LGTM

**Pull request merged**